geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/Stat.scala

```diff
@@ -79,7 +82,14 @@ trait Stat {
    *
    * @return stat as a json string
    */
-  def toJson: String
+  def toJson: String = Stat.JSON.toJson(toJsonObject)
+
+  /**
+    * Returns a representation of the stat to be serialized
+    *
+    * @return stat as a json serializable object
+    */
+  def toJsonObject: Any
```

Comment:
maybe just add that to the scala docs. Presumably in practice it would have to be either a Map or a List to create a valid json object